src/components/FrontPage/WelcomeBox.js

```diff
@@ -8,11 +8,17 @@ import {getTranslator} from '../../selectors/translate';
 import LinkContainer from 'react-router-bootstrap/lib/LinkContainer';
 import {setWelcomeBox} from '../../action_creators';
 
-const WelcomeBox = ({t, welcomeBox, setWelcomeBox}) => {
-  const startButton = <div className={styles.center}>
-    <LinkContainer to={t('frontpage.welcomebox.buttonlink')}>
+const WelcomeBox = ({t, welcomeBox, setWelcomeBox, checkboxes}) => {
+  //TODO: This button should send the user to the lesson where the last checkbox was checked.
+  //If every checkbox in that lesson is checked the button should send the user to the course of the last lesson
+  const bigButton = <div className={styles.center}>
+    <LinkContainer to={checkboxes === {}
```

Comment:
Hvorfor gjør du `checkboxes === {}` for å sjekket om man skal "starte" eller "fortsette"? Burde det ikke være en egen state/variabel som fortalte deg hvilken lesson du var inne på sist, uavhengig av checkbox? Hva om du var inne på en lesson som ikke hadde checkboxes? Eller om du gjorde noe som ikke krevede checkboxing?